src/course/syllabus/application/feature/project.md

```diff
@@ -0,0 +1,49 @@
+Build a feature to include on your application website. This project is open-ended and a chance for you to explore creating something with different complex parts that interact with each other.
```

Comment:
Also are they expected to include this feature on their application site? Or will it be a separate repo? Since they're building it beforehand it will probably have to be in its own repo—do we expect them to copy/paste it into the new website repo later?

Reply:
Do you think it'd make sense to swap the projects around? They'd then have the context of the website and could build this on top of it.

For the 6 projects, I imagine the application website can feature the project gallery and feature. The fruit shop, comment box, and tribute page are probably in separate repo's on separate sites.